### ISSUE TITLE
Unable to change the default reconciliation time for all the resources in Crossplane

### ISSUE THREAD

**Issue by vallabhsonar** [crossplane/crossplane#5934]
Hi, 

While testing the reconcile functionality of Crossplane for AWS resources, I observed that the reconciliation process took exactly 1 minute for resources like VPCs, Security Groups, EC2 instances, and route tables. However, for resources such as S3 buckets and network interfaces, it consistently took 5 minutes. 

I reviewed the documentation [here](https://docs.crossplane.io/latest/concepts/pods/) to modify this default behavior by setting a specific interval of 1 minute for all resources. Despite editing the Crossplane deployment as below with the appropriate values, the changes are not working as expected.

1. using --poll-interval
$ kubectl edit deployment crossplane --namespace crossplane-system
apiVersion: apps/v1
kind: Deployment
spec:
  template:
    spec:
      containers:
      - args:
        - core
        - start
        - --poll-interval=1m
        
2. using --enable-realtime-compositions
$ kubectl edit deployment crossplane --namespace crossplane-system
apiVersion: apps/v1
kind: Deployment
spec:
  template:
    spec:
      containers:
      - args:
        - core
        - start
        - --enable-realtime-compositions

### What environment did it happen in?
Crossplane version: v1.16

* Cloud provider or hardware configuration: AWS
* Kubernetes version (use `kubectl version`): v1.30.3
* OS (e.g. from /etc/os-release): Ubuntu 20.04.6

**Comment by jbw976** [comment]
Can you define exactly what you mean by "reconciliation time"? I think `--poll-interval` is what you're looking for, but I'm not 100% sure 🙇‍♂️ 

Did you try setting these values on the provider instead? That's the one doing the work to reconcile these resources with AWS, so I think the setting should be going there. See https://docs.crossplane.io/latest/concepts/providers/#runtime-configuration

**Comment by vallabhsonar** [comment]
Hi @jbw976,

To clarify with an example: I created a VPC on AWS, and when I deleted it through the AWS console, the deletion was reconciled according to the provider's default timing. Now, I would like to modify this timing using `--poll-interval `or another interval option.

Additionally, as noted in the [documentation](https://docs.crossplane.io/latest/concepts/providers/#runtime-configuration), I attempted to set `--poll-interval` in a `DeploymentRuntimeConfig`, but it still hasn't updated the reconcile time as specified.

![image](https://github.com/user-attachments/assets/9a45bc3d-8086-4ca2-82fe-8dcf44328ba9)

**Comment by jbw976** [comment]
Hmm, do you see that value making it all the way to the provider-aws-ec2 pod spec? 🤔

**Comment by vallabhsonar** [comment]
Yes @jbw976, the value updates without any errors. Could we schedule a meeting with you or your team to discuss this issue further?

**Comment by Aym3nTN** [comment]
I had a similar case with s3 provider. I had provided `--help` to get the list of args (couldn't find it on the internet, if someone can redirect me to a doc, I'd be grateful) 
![2024-10-05_18-23-20](https://github.com/user-attachments/assets/2ce797c9-3fb5-4540-9377-4394bdf1ee24)

So the correct arg is `--poll=1m`
